microsoft/testsuites/mdatp/check-mdatp.sh

```diff
@@ -0,0 +1,29 @@
+#! /bin/sh
+
+# check the following folders for printable files, print them
+FOUND_DEFENDER=1
+MDATP_OPT_DIR='/etc/opt/microsoft/mdatp/'
+MDATP_EXTENSION_DIR='/var/lib/waagent/Microsoft.Azure.AzureDefenderForServers.MDE.Linux'
```

Comment:
The extension directory is startsWith "/var/lib/waagent/Microsoft.Azure.AzureDefenderForServers.MDE.Linux"
The real path is /var/lib/waagent/Microsoft.Azure.AzureDefenderForServers.MDE.Linux-1.X.X.X" (version)

(MDE.Linux extension owner)

Reply:
Ahhhhhh thank you!!

Reply:
I was wondering why we hadn't seen any of these yet 😜

Reply:
fixed, using globs to grab whatever is there.